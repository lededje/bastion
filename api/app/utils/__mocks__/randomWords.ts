export default (num: number) => new Array(num).fill('MOCKED_RANDOM_WORD')