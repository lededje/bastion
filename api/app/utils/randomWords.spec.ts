import randomWords from './randomWords';

describe('randomWords', () => {
  beforeEach(() => {
    jest.spyOn(global.Math, 'random').mockReturnValue(0.123456789);
  });

  afterEach(() => {
      jest.spyOn(global.Math, 'random').mockRestore();
  })

  it('returns three random words', () => {    
    expect(randomWords(3)).toMatchSnapshot();
  })
})
