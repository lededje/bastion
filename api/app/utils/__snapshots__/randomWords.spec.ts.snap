// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`randomWords returns three random words 1`] = `
Array [
  "buzz",
  "buzz",
  "buzz",
]
`;
